packaging/deb/init.d/grafana-server

```diff
@@ -57,85 +57,85 @@ if [ -r /etc/default/rcS ]; then
 fi
 
 # overwrite settings from default file
-if [ -f "$DEFAULT" ]; then
-	. "$DEFAULT"
+if [ -f "${DEFAULT}" ]; then
+	. "${DEFAULT}"
 fi
 
 DAEMON_OPTS="--pidfile=${PID_FILE} --config=${CONF_FILE} cfg:default.paths.data=${DATA_DIR} cfg:default.paths.logs=${LOG_DIR} cfg:default.paths.plugins=${PLUGINS_DIR}"
 
 case "$1" in
   start)
 
-	log_daemon_msg "Starting $DESC"
+	log_daemon_msg "Starting ${DESC}"
 
-	pid=`pidofproc -p $PID_FILE grafana`
-	if [ -n "$pid" ] ; then
+	pid=`pidofproc -p ${PID_FILE} grafana`
```

Comment:
useless; also, as long as accent gravis-style command substitution is used, this cannot ever be secured; something like…

    pid=$(pidofproc -p "$PID_FILE" grafana)

… would work.